scenarios/sqs_flag_shop/terraform/iam.tf

```diff
@@ -0,0 +1,131 @@
+resource "aws_iam_user" "cg-web-sqs-manager" {
+  name = "cg-web-sqs-manager-${var.cgid}"
+  tags = {
+    Name     = "cg-web-sqs-manager-${var.cgid}"
+    Stack    = var.stack-name
+    Scenario = var.scenario-name
+  }
+}
+
+resource "aws_iam_access_key" "cg-web-sqs-manager_access_key" {
+  user = aws_iam_user.cg-web-sqs-manager.name
+}
+
+resource "aws_iam_policy_attachment" "user_SQS_full_access" {
+  name       = "SQSFullAccessAttachment"
+  policy_arn = "arn:aws:iam::aws:policy/AmazonSQSFullAccess"
+  users      = [aws_iam_user.cg-web-sqs-manager.name]
+}
+
+resource "aws_iam_user" "cg-sqs-user" {
+  name = "cg-sqs-user-${var.cgid}"
+  tags = {
+    Name     = "cg-sqs-user-${var.cgid}"
+    Stack    = var.stack-name
+    Scenario = var.scenario-name
+  }
+}
+
+resource "aws_iam_access_key" "cg-sqs-user_access_key" {
+  user = aws_iam_user.cg-sqs-user.name
+}
+
+resource "aws_iam_role_policy" "cg-sqs_scenario_policy" {
+  name = "cg-sqs_scenario_policy"
+  role = aws_iam_role.cg-sqs_send_msg_role.id
+
+  policy = jsonencode({
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Sid" : "VisualEditor0",
+        "Effect" : "Allow",
+        "Action" : [
+          "iam:Get*",
+          "iam:List*"
+        ],
+        "Resource" : aws_sqs_queue.cg_cash_charge.arn
+      }
+    ]
+  })
+}
+
+resource "aws_iam_user_policy" "cg-sqs_scenario_assumed_role_policy" {
+  name   = "cg-sqs-scenario-assumed-role-policy"
+  user   = aws_iam_user.cg-sqs-user.name
+
+  policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [
+      {
+        Sid    = "VisualEditor0",
+        Effect = "Allow",
+        Action = [
+          "iam:Get*",
+          "iam:List*",
+        ],
+        Resource = "*",
+      },
+      {
+        Sid    = "VisualEditor1",
+        Effect = "Allow",
+        Action = "sts:AssumeRole",
+        Resource = aws_iam_role.cg-sqs_send_msg_role.arn,
+      },
+    ],
+  })
+}
+
+resource "aws_iam_role" "cg-sqs_lambda_role" {
+  name = "cg-sqs_rds_lambda_role"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": [
+          "lambda.amazonaws.com"
+        ]
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role" "cg-sqs_send_msg_role" {
+  name = "cg-sqs_send_msg_role"
+
+  assume_role_policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "${aws_iam_user.cg-sqs-user.arn}"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {}
+        }
+    ]
+}
+EOF
+}
+
+locals {
+  sqs_lambda_role_policy_arns = [
+    "arn:aws:iam::aws:policy/AmazonSQSFullAccess",
+    "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole",
```

Comment:
Needs to be `arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole` instead of AWSLambdaBasicExecutionRole


Getting the following error

> Error: creating Lambda Function (lambda_function): operation error Lambda: CreateFunction, https response error StatusCode: 400, RequestID: 102bca60-bfde-4e6e-82d8-fac3018967e8, InvalidParameterValueException: The provided execution role does not have permissions to call CreateNetworkInterface on EC2